test/libsolidity/SemanticTest.cpp

```diff
@@ -295,6 +295,10 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 
 bool SemanticTest::checkGasCostExpectation(TestFunctionCall& _test, bool _compileViaYul) const
 {
+	// TODO: check other evm versions
+	if (m_evmVersion != EVMVersion::istanbul())
```

Comment:
Why not always latest?

Reply:
I was running default setup for isoltest and it was istanbul for me (is it dependant on which version I have installed?).
I can do latest in any case.

Reply:
Is there a way to get the latest one other than `EVMVersion{}`? Something like `EVMVersion::latest()` would be more readable.

Reply:
@mijovic 
> (is it dependant on which version I have installed?).

I think `istanbul` is the default now and you can choose a different one using the `--evm-version` option:

```
soltest -- --evm-version istanbul
isoltest --evm-version istanbul
```

In CI we're running tests on all versions.

Reply:
Changed to latest

Reply:
I asked about `berlin` on gitter and here latest should be `istanbul`. It's the latest stable version. So this should be set to `EVMVersion{}`, not `EVMVersion::latest()`.

In that case I wonder if it made sense to add the `latest()` helper after all. It looks like the one you would want by default but that's misleading. I originally assumed that latest and default would be pretty much the same thing for us but they're not.

Reply:
Removed latest helper and run only on istanbul now

Reply:
It's `istanbul` now which is good but I maybe I wasn't clear that I think you should actually use `EVMVersion{}` (i.e. using the default constructor), not `EVMVersion::istanbul()`. This way we won't have to find and update this when we switch the default in that class. [That's what's used for the `--evm-version` option](https://github.com/ethereum/solidity/blob/v0.8.1/test/Common.cpp#L182-L193).

Reply:
This still needs to be addressed, right?